db/migrate/20221020121609_add_precis_and_slug_to_news_items.rb

```diff
@@ -0,0 +1,10 @@
+Sequel.migration do
+  change do
+    alter_table :news_items do
+      add_column :precis, String
```

Comment:
I have no idea what a precis is. Is there something from common vernacular that we might use for this?

Reply:
Okay, googled this. Could we use abstract/summary, instead?